settings.gradle.kts

```diff
@@ -19,7 +19,26 @@ plugins {
 
 rootProject.name = "hedera-services"
 
-// Define the subprojects
+val services = ArrayList<String>()
+services.add("hedera-admin-service")
+services.add("hedera-consensus-service")
+services.add("hedera-file-service")
+services.add("hedera-network-service")
+services.add("hedera-scheduled-service")
+services.add("hedera-smart-contract-service")
+services.add("hedera-token-service")
+services.add("hedera-util-service")
+
+services.forEach { s ->
+    run {
+        include(":" + s)
+        project(":" + s).projectDir = file("modules/" + s)
+        val implProjectName = s + "-impl"
+        include(":" + implProjectName)
+        project(":" + implProjectName).projectDir = file("modules/" + implProjectName)
+    }
+}
+
```

Comment:
What is the effect of this? I expected to see `include(":hedera-node:modules:hedera-file-service")` or something along those lines

Reply:
i switched it to the basic version. We should discuss this later.  By doing it like that the "modules" folder becomes a gradle module that has tasks like compileJava and so on. I tried to avoid that and played with gradle.